src/Components/ViewSingleArticle.jsx

```diff
@@ -24,6 +24,20 @@ const ViewSingleArticle = (props) => {
     <div>posted {DateConverter(singleArticle.created_at)}</div>
     <span>{singleArticle.votes} votes</span>
     <div>{singleArticle.comment_count} comments</div>
+    <ul> 
```

Comment:
this can be its own comments component which rendered as a direct child of the single article component - we can cut out the middle man (the view single article component)